frontend/src/components/common/LogRecordsTable.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import {
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from "@chakra-ui/react";
+import { LogRecord } from "../../types/LogRecordTypes";
+
+type Props = {
+  logRecords: LogRecord[];
+};
+
+const LogRecordsTable = ({ logRecords }: Props): React.ReactElement => {
+  return (
+    <div>
+      <TableContainer marginTop="12px" height="70vh" overflowY="scroll">
+        <Table
+          variant="simple"
+          style={{ minHeight: "400px", verticalAlign: "middle" }}
+        >
+          <Thead className="table-header">
+            <Tr>
+              <Th>Date</Th>
+              <Th>Time</Th>
+              <Th>Resident</Th>
+              <Th>Note</Th>
+              <Th>Employee</Th>
+              <Th>Attn To</Th>
+            </Tr>
+          </Thead>
+
+          <Tbody>
+            {/* TODO: replace mockRecords with logRecords */}
+            {logRecords.map((record) => {
+              // TODO: Investigate alternative methods for date storage + creation
+              const dateObj = new Date(record.datetime);
+
+              return (
+                <Tr key={record.logId} style={{ verticalAlign: "middle" }}>
+                  <Td width="5%">
+                    {`${dateObj.getMonth()} ${dateObj.getDate()} ${dateObj.getFullYear()}`}
+                  </Td>
+                  <Td width="5%">
+                    {`${dateObj.getHours()}:${dateObj.getMinutes()}`}
```

Comment:
on the Figma, we use 12-hour time (ex: 6:45pm) instead of 24-hour time (ex: 18:45).

Reply:
Resolved 🕐